src/Github/Domain/Value/Label.php

```diff
@@ -21,20 +21,37 @@
 final class Label
 {
     private string $value;
+    private ?string $color;
 
-    private function __construct(string $value)
+    private function __construct(string $value, ?string $color = null)
     {
         $value = trim($value);
         Assert::stringNotEmpty($value);
 
+        if (null !== $color) {
+            Assert::stringNotEmpty($color);
+        }
+
         $this->value = $value;
+        $this->color = $color;
     }
 
     public static function fromString(string $value): self
     {
         return new self($value);
     }
 
+    public static function fromResponse(array $response): self
```

Comment:
Please document any `array` parameter (so here it would be `array{name: string, color: string}`?)

Reply:
What about use jms_serializer and deserialize string directly from github API?

Reply:
That could make sense